src/pybind/nnet3/nnet_example_pybind.cc

```diff
@@ -40,4 +41,19 @@ void pybind_nnet_example(py::module& m) {
             "SparseMatrix would be the natural format for posteriors).");
     // TODO(fangjun): other constructors, fields and methods can be wrapped when
   }
+  {
+    using PyClass = NnetExample;
+    py::class_<PyClass>(m, "NnetExample")
```

Comment:
You can copy and paste the documentation from c++ to pybind.

For example,

```cpp
py::class_<PyClass>(m, "NnetExample", "NnetExample is the input data....")
```

Later, we will generate static HTML documentation from these comments.

Reply:
I'll add it